ab_tool/canvas.py

```diff
@@ -92,12 +90,41 @@ def _get_installed_intervention_point_urls(self):
         return installed_intervention_point_urls
 
 
-def get_unsorted_students(request, experiment):
+def get_unassigned_students(request, experiment):
+    """ Gets the list of students that have not been assigned a track.
+        This version of the function is for requests that have come through
+        a web view and consequently have a django request object off of which
+        to make a canvas request context """
     request_context = get_canvas_request_context(request)
-    course_id = get_lti_param(request, "custom_canvas_course_id")
+    return get_unassigned_students_with_context(request_context, experiment)
+
+
+def get_unassigned_students_with_stored_credentials(course_object, experiment):
```

Comment:
the couse object is not an object to represent the course, but rather notifications about the course.  we have no guarantee that the course_noficiation object actually exists.  it has been renamed.  passing it as an argument here eliminates a redundant database call